python/ray/_private/usage/usage_lib.py

```diff
@@ -244,10 +244,17 @@ def _put_library_usage(library_usage: str):
 class TagKey(Enum):
     _TEST1 = auto()
     _TEST2 = auto()
+
+    # RLlib
     RLLIB_FRAMEWORK = auto()
     RLLIB_ALGORITHM = auto()
     RLLIB_NUM_WORKERS = auto()
 
+    # Serve
+    SERVE_USING_V1_API = auto()
+    SERVE_USING_V2_API = auto()
+    SERVE_NUM_DEPLOYMENTS = auto()
```

Comment:
Please add descriptions of what this is measuring. (cc @thomasdesr @pcmoritz we shouldn't approve metrics without clear descriptions).